src/ResourceManagement/Compute/ComputeManagement/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.11.0-Nightly20150907
+set autoRestVersion=0.11.0-Nightly20150911
```

Comment:
@markcowl @stankovski 
Do you have any more lateset version that you would recommend us to use? :-)